removeElement/index.js

```diff
@@ -0,0 +1,24 @@
+window.addEventListener('DOMContentLoaded', () => {
+    const array = [1, 2, 3, 4, 5, 6, 5, 7];
+
+    // Dirty
+
+    function removeElementDirty(arr, element) {
+        while (arr.findIndex((el) => el === element) !== -1) {
```

Comment:
why do you need `while` here?

Reply:
in case we have more than one element in array, that need to be deleted